test/rng_testsuite/conformance_tests/minstd_rand_rand0_test.pass.cpp

```diff
@@ -27,6 +27,9 @@
 //     The 10000th consecutive invocation of a default-constructed object of type minstd_rand
 //     produces the value 399268537
 
+#include <oneapi/dpl/execution>
+#include "support/pstl_test_config.h"
```

Comment:
Does it make sense to rename pstl_test_config.h to test_config.h? As it is also included in RNG tests

Reply:
Yes, this make sense. I have created issue.